test/integration/array_test.ts

```diff
@@ -614,19 +650,30 @@ describe('Array', function () {
 
       d1.update((root) => {
         const prev = root['k1'].getElementByIndex(3);
+
```

Comment:
minor: we can remove this empty string :)
https://github.com/yorkie-team/yorkie-js-sdk/pull/229/files#diff-8a241fd6d525dc74d88294517006d18e60e19275b5577e0399ebdbbe5a639599R655

https://github.com/yorkie-team/yorkie-js-sdk/pull/229/files#diff-8a241fd6d525dc74d88294517006d18e60e19275b5577e0399ebdbbe5a639599R662

https://github.com/yorkie-team/yorkie-js-sdk/pull/229/files#diff-8a241fd6d525dc74d88294517006d18e60e19275b5577e0399ebdbbe5a639599R664